homeassistant/components/spotify/media_player.py

```diff
@@ -266,7 +290,7 @@ def shuffle(self) -> bool:
     def supported_features(self) -> int:
         """Return the media player features that are supported."""
         if self._me["product"] != "premium":
-            return 0
+            return SUPPORT_SPOTIFY  # SUPPORT_BROWSE_MEDIA | SUPPORT_SELECT_SOURCE
```

Comment:
Commented code ?

Reply:
That was accidental. I'll fix that after lunch.